internal/flow/notify.go

```diff
@@ -94,11 +96,12 @@ func parseCommitMessage(commitMessage string) (FluxReleaseMessage, error) {
 	if len(matches) < 1 {
 		return FluxReleaseMessage{}, errors.New("not enough matches")
 	}
-
+	fmt.Printf("matches: %v", matches)
```

Comment:
Debug? 🙈 Might be nice to keep though as a formatted log line?

Reply:
🤦